cfe_internal/enterprise/ha/ha_update.cf

```diff
@@ -0,0 +1,129 @@
+bundle agent ha_update
+{
+ vars:
+      "master_hub_location" string => "/var/cfengine/state/master_hub.dat";
+      "hubs_keys_location" string => "/var/cfengine/ppkeys_hubs"; 
+      "ppkeys_staging" string => "/var/cfengine/ppkeys_staging";
+ methods:
+  enable_cfengine_enterprise_hub_ha.am_policy_hub::
+      "share_keys" usebundle => ha_share_hub_keys;
+      "sync_hub_data" usebundle => ha_hub_sync;
+
+  enable_cfengine_enterprise_hub_ha::
+      "sync_agent_data" usebundle => ha_agent_sync;
+
+  enable_cfengine_enterprise_hub_ha.hub_data_synced::
```

Comment:
This will be activated only if 'hub_data_synced' is set. It can be set only in 'bundle agent ha_hub_sync' which is hub specific.